pkg/handler/collector/github/github_test.go

```diff
@@ -0,0 +1,174 @@
+//
+// Copyright 2022 The GUAC Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package github
```

Comment:
should be flagged as an integration test for now (example: https://github.com/guacsec/guac/blob/fa6efbb9b2313a4808dd6fc5030d144a3e734a12/pkg/assembler/graphdb/graphdb_test.go#L16)

I’d say another improvement could be mocking up the `RepositoryService` as part of the client so we can test this without relying on github, but may not be necessary in the short term, so i wouldn’t worry about it unless you want to.

Reply:
^